File: answers/Go/@codehakase/02-lottery.go
package main

import (
	"bufio"
	"fmt"
	"math/rand"
	"os"
	"time"
)

func main() {
	rand.Seed(time.Now().UnixNano())
	var a, b, c int
	for {
		reader := bufio.NewReader(os.Stdin)
		fmt.Print("Press Enter key to continue...")
		_, _ = reader.ReadString('\n')
		a = rand.Intn(10)
		b = rand.Intn(10)
		c = rand.Intn(10)
		fmt.Printf("%d %d %d\n", a, b, c)
		if a == 7 || b == 7 || c == 7 {
			fmt.Println("Congratulations!!\n")
		} else {
			fmt.Println("Try again! Better luck next time.\n")
		}
	}
}

File: programming/answers/C/@mykeels/02-lottery.c
#include <stdio.h>
#include <time.h>
#include <stdlib.h>

int main() {
    srand(time(NULL));

    while (1) {
        printf("Press any key: ");
        getchar();

        int a = rand() % 10;
        int b = rand() % 10;
        int c = rand() % 10;

        printf("%d %d %d\n", a, b, c);

        if (a == 7 || b == 7 || c == 7) {
            printf("Congratulations!\n");
        }
        else {
            printf("Try again! Better luck next time.\n");
        }
    }
    
    return 0;
}